lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,60 @@
+require 'httparty'
+
+class EdamamApiWrapper
+  BASE_URL = "https://api.edamam.com/search?"
+  ID = ENV["RECIPE_ID"]
+  TOKEN = ENV["RECIPE_TOKEN"]
+  TOKENS_URL = "&app_id=#{ID}&app_key=#{TOKEN}"
```

Comment:
A URL like this will only give you 10 responses, if you include `&to=100` you'll get 100 responses.